styles.css

```diff
@@ -0,0 +1,108 @@
+body {
+  font-family: sans-serif;
+  margin: 0;
+}
+
```

Comment:
* {
  box-sizing: border-box;
  padding: 0;
  margin: 0;
}
- Kindly try to use on top of your stylesheet to git rid of the extra by default space around your webpage so your design become better